skrifa/src/outline/cff/hint.rs

```diff
@@ -1386,6 +1389,106 @@ mod tests {
         }
     }
 
+    #[test]
+    fn hint_map_midpoint_bug() {
```

Comment:
Can we make a test of just overflow with less setup noise? - this is a slightly stunning amount of setup noise and seemingly tests rather more than just the actual bug.

Reply:
I can just remove the test. Looking at it now, I think the fix is self evident and there’s no way to poke directly at that code without the setup.

Reply:
That seems like a yellow flag in it's own right. What prevents us from making this code more readily testable?

Reply:
Your comment got me thinking so I changed the test to synthesize the "initial" hint map (which is required to trigger this code path) rather than the blues and then call the offending `insert` function directly. Reduces the setup noise quite a bit. wdyt?